docs/sts/README.md

```diff
@@ -11,15 +11,15 @@ Following are advantages for using temporary credentials:
 - Temporary credentials have a limited lifetime, there is no need to rotate them or explicitly revoke them. Expired temporary credentials cannot be reused.
 
 ## Identity Federation
-[**Client grants**](./client-grants.md) - Let applications request `client_grants` using any well-known third party identity provider such as KeyCloak, WSO2. This is known as the client grants approach to temporary access. Using this approach helps clients keep Minio credentials to be secured. Minio STS client grants supports WSO2, Keycloak.
+[**Client grants**](https://github.com/minio/minio/blob/master/docs/sts/client-grants.md) - Let applications request `client_grants` using any well-known third party identity provider such as KeyCloak, WSO2. This is known as the client grants approach to temporary access. Using this approach helps clients keep Minio credentials to be secured. Minio STS client grants supports WSO2, Keycloak.
```

Comment:
```suggestion
[**Client grants**](https://github.com/minio/minio/blob/master/docs/sts/client-grants.md) - Let applications request `client_grants` using any well-known third party identity provider such as KeyCloak, WSO2. This is known as the client grants approach to temporary access. Using this approach helps clients keep Minio credentials secure. Minio STS client grants support WSO2, Keycloak.
```

Reply:
Would we not support and STS client grants. Why only WSO2 and keycloak mentioned here?

Reply:
Just what was tested ..

Reply:
can we say Minio STS has been tested with WSO2 and KeyCloak client grants?

Reply:
We can say in a new PR. This PR is to only fix links.

Reply:
Ok someone will be sending a comprehensive PR for english separately.